### ISSUE TITLE
Amazfit Bip Caller ID not working in versions 0.32.2+

### ISSUE THREAD

**Issue by sobrus** [Freeyourgadget/Gadgetbridge#1481]
Since version 0.32.2 caller id on Amazfit Bip is displayed only for fraction of a second. Then it disappears. Downgrading Gadgetbridge to version 0.32.1 fixes the problem.

Device: Amazfit Bip, firmware 1.1.5.16 (also tested on 1.1.5.02), HW V0.11.2.4
Android version 6.0 MRA58K, MIUI 10.2.1.0 MBFMIXM Global stable.

**Comment by ashimokawa** [comment]
Can you try to disable VoIP call support in settings?

**Comment by sobrus** [comment]
Yes, I've just checked and disabling VoIP call support fixes the issue too.
